jumanji/environments/packing/knapsack/env.py

```diff
@@ -96,14 +98,16 @@ def __init__(
                 transition. The function must compute the reward based on the current state,
                 the chosen action, the next state and whether the action is valid.
                 Implemented options are [`DenseReward`, `SparseReward`]. Defaults to `DenseReward`.
+            viewer: `Viewer` used for rendering. Defaults to `KnapsackViewer`.
             render_mode: string that defines the mode of rendering.
```

Comment:
We probably should not have the choice of the render mode here, it seems to be mixing composite and strategy pattern. We just want the `env_viewer` that defaults to some `render_mode` right?